engine/clients/redis/search.py

```diff
@@ -64,7 +64,7 @@ def search_one(cls, vector, meta_conditions, top) -> List[Tuple[int, float]]:
         params_dict = {
             "vec_param": np.array(vector).astype(np.float32).tobytes(),
             "K": top,
-            "EF": cls.search_params["search_params"]["ef"],
+            "EF": cls.search_params["params"]["ef"],
             **params,
```

Comment:
```suggestion
            **params,
```
Can we change `redis-single-node.json` to directly use `EF` instead of `ef`?

Reply:
Did you mean `**cls.search_params["config"],` instead of `"EF": cls.search_params["params"]["ef"],`? In that case I don't think so, as case matter, redis doesn't recognize 'ef' instead of 'EF', and I don't want to change the case in redis-single-node.json

Reply:
> Did you mean using **cls.search_params["config"], instead of "EF": cls.search_params["params"]["ef"]

Yes. 

> I don't want to change the case in redis-single-node.json

My idea was to give full control to the configs. If `EF` is what redis accepts, it should go in the config. 
Otherwise, Redis would be inconsistent with other engines. I'm fine with either.